tests/ui/unnecessary_wraps.stderr

```diff
@@ -97,10 +97,62 @@ help: remove `Option` from the return type...
    |
 LL |     fn func12() -> i32 {
    |                    ^^^
-help: ...and change the returning expressions
+help: ...and then change the returning expressions
    |
 LL |         1
    |
 
-error: aborting due to 5 previous errors
+error: this function's return value is unnecessary
+  --> $DIR/unnecessary_wraps.rs:120:1
+   |
+LL | / fn issue_6640_1(a: bool, b: bool) -> Option<()> {
+LL | |     if a && b {
+LL | |         return Some(());
+LL | |     }
+...  |
+LL | |     }
+LL | | }
+   | |_^
+   |
+help: remove the return type...
+   |
+LL | fn issue_6640_1(a: bool, b: bool) -> Option<()> {
+   |                                      ^^^^^^^^^^
+help: ...and then change the returning expressions
+   |
+LL |         return ;
+LL |     }
+LL |     if a {
+LL |         Some(());
+LL |         
+LL |     } else {
+ ...
+
+error: this function's return value is unnecessary
+  --> $DIR/unnecessary_wraps.rs:133:1
+   |
+LL | / fn issue_6640_2(a: bool, b: bool) -> Result<(), i32> {
+LL | |     if a && b {
+LL | |         return Ok(());
+LL | |     }
+...  |
+LL | |     }
+LL | | }
+   | |_^
+   |
+help: remove the return type...
+   |
+LL | fn issue_6640_2(a: bool, b: bool) -> Result<(), i32> {
+   |                                      ^^^^^^^^^^^^^^^
+help: ...and then change the returning expressions
```

Comment:
About the new message, you are right, it would be more clear.
I am, however, not sure if I should hard-code the `"Some"` and `"Ok"` strings. I could get the text from the actual return expressions in the body of the function, but we never know how the user could format those.
Is there any alternative to hard-coding and this?

Reply:
I think hard-coding them would be fine since it is just `Some` and `Ok`. I'd also be fine with taking that part out of the message like "remove the returned values".